src/xenia/base/string_util.h

```diff
@@ -66,7 +66,7 @@ inline std::string to_hex_string(const __m128& value) {
   char buffer[128];
   float f[4];
   _mm_storeu_ps(f, value);
-  std::snprintf(
+  snprintf(
```

Comment:
I don't think this belongs to this PR. Why did you drop `std::`?

Reply:
Clarified the drop of std:: in the commit message, copying here:
This seems to be an issue with Microsoft declaring a snprintf macro in an included header which causes issues.
Related issues:

nlohmann/json#1408
kodi-pvr/pvr.vuplus#162